js/app.js

```diff
@@ -0,0 +1,145 @@
+document.addEventListener('DOMContentLoaded', function () {
+
+  /* ---------------------------------------------------- */
+  /* ----- SLIDER (SLIDESHOW) IMAGES ----- */
+  /* ---------------------------------------------------- */
+
+  let slideIndex = 0;
+
+  function showSlide(index) {
+    let slides = document.querySelectorAll('.slider__slide');
+
+    if (index >= slides.length) {
```

Comment:
to dość prosty warunek więc możesz użyć składni `a>b ? index=0 : index=1;` - ternary operator